HISTORY.rst

```diff
@@ -2,6 +2,25 @@
 History
 =======
 
+3.5.0 (2020-01--15)
+-------------------
+
+Changes:
+
+* Bulk data mappings now allow bulk_mode: Serial to select serial mode.
```

Comment:
```suggestion
* The ``load_dataset`` task now accepts a ``bulk_mode`` option which can be set to ``Serial`` to load batches serially instead of in parallel.
```